x-pack/legacy/plugins/uptime/index.ts

```diff
@@ -30,6 +31,7 @@ export const uptime = (kibana: any) =>
         main: 'plugins/uptime/app',
         order: 8900,
         url: '/app/uptime#/',
+        mappings: savedObjectMappings,
```

Comment:
shouldn't it be uiExports.savedObjectMappings instead?

Reply:
Ah, thanks!